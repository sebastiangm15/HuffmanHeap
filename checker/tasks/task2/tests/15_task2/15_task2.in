12
15 CVF48
21 KMA73
26 VXE76
32 TIN43
49 LCS44
51 XQX45
52 BQL46
53 EWN25
78 NVV52
81 EUM94
85 AJP78
94 UFT53
1
0101011001000000111110
