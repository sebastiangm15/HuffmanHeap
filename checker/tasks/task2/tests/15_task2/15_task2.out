NVV52 LCS44 VXE76 EWN25 TIN43 XQX45 
